htdocs/feedback_mri_popup.php

```diff
@@ -47,6 +47,9 @@
     $comments->setPredefinedComments($_POST['savecomments']['predefined']);
 
     // save all textual comments but only if there is an entry [sebas]
+    // The foreach below is operating on an array even though phan doesn't
```

Comment:
this seems hacky? why would phan think it was not an array if it was and if the rule is supposed to check exactly that?

Reply:
would defining an empty array variable, and then setting it to `$_POST['savecomments']['text']` bypass this?

Reply:
I'll give it a shot!

Reply:
No 😞

Reply:
:(

Reply:
We should always be using the most up-to-date version of phan (or just about any other library) that doesn't cause breakages in Travis..

With regard to this, I don't like the suppression either. Why can't you use `\Utility::asArray` to ensure foreach is working with an array?

Reply:
I'll give it a try

Reply:
Passes phan! Not really sure how to test it though

Reply:
Click on image level feedback in the imaging browser and make sure things still work/save in the popup.

Reply:
Found some type errors while testing. Fixed in #4285. Will block this until that's merged.

Reply:
the comment on these lines can be removed now no?